src/deck.py

```diff
@@ -1,7 +1,20 @@
 from card import Card
-
+import random
+import uuid
 
 class Deck:
+    suits = ['Hearts', 'Diamonds', 'Clubs', 'Spades']
+    values = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'Jack', 'Queen', 'King', 'Ace']
```

Comment:
Same issue here. This will almost certainly cause the UI to fail unless you've modified it accordingly (which you haven't done in this Pull Request).